src/components/bs5/banner/banner.hbs

```diff
@@ -24,6 +24,8 @@
         </div>
     </div>
 
-    <div class="banner-image {{image.classes}}" style="--banner-background-img:url({{image.url}})"></div>
+    {{#if image }}
+    <div class="banner-image {{image.classes}}" style="background-image:url({{ image.url }})"></div>
```

Comment:
was this meant to be part of this pr, as its unrelated to the QGDS-21
QGDS CICD, Config, Environments, Build, Release

also should this be a child task to the epic?